YapDatabase/Extensions/RTreeIndex/YapDatabaseRTreeIndexTransaction.h

```diff
@@ -55,23 +55,6 @@ NS_ASSUME_NONNULL_BEGIN
 **/
 - (BOOL)getNumberOfRows:(NSUInteger *)count matchingQuery:(YapDatabaseQuery *)query;
 
-/**
- * This method assists in performing a query over a subset of rows,
- * where the subset is a known set of keys.
- *
- * For example:
- *
- * Say you have a known set of items, and you want to figure out which of these items fit in the rectangle.
- *
- * NSArray *keys = [self itemKeys];
- * NSArray *rowids = [[[transaction ext:@"idx"] rowidsForKeys:keys inCollection:@"tracks"] allValues];
- *
- * YapDatabaseQuery *query =
- *   [YapDatabaseQuery queryWithFormat:@"WHERE minLon > 0 AND maxLat <= 10 AND rowid IN (?)", rowids];
- **/
-- (NSDictionary<NSString*, NSNumber*> *)rowidsForKeys:(NSArray<NSString *> *)keys
```

Comment:
I removed some methods we never use.

Reply:
Hm, even though we're getting away from it, I'd think we would want to minimize our upstream diff wherever possible. e.g. there are some patches from yap/master I still might like to pull in and this will make it ever more difficult.

Reply:
Okay, I've reverted these deletes.  The intent was to be certain that all YDB enumerations were batched without converting the enumerations we don't use.